cloudinit/config/cc_apt_configure.py

```diff
@@ -6,225 +6,364 @@
+                        dictionary which must have an entry for
+                         ``arches``, specifying which architectures
+                        that config entry is for. The keyword
+                         ``default`` applies to any architecture not
+                        explicitly listed. The mirror url can be specified
+                        with the ``uri`` key, or a list of mirrors to
+                        check can be provided in order, with the first
+                        mirror that can be resolved being selected. This
+                        allows the same configuration to be used in
+                        different environment, with different hosts used
+                        for a local apt mirror. If no mirror is provided
+                        by ``uri`` or ``search``, ``search_dns`` may be
+                        used to search for dns names in the format
+                         ``<distro>-mirror`` in each of the following:
+
+                            fqdn of this host per cloud metadata,
+                            localdomain,
+                            domains listed in ``/etc/resolv.conf``.
+
+                        If there is a dns entry for ``<distro>-mirror``,
+                        then it is assumed that there is a distro mirror
+                        at ``http://<distro>-mirror.<domain>/<distro>``.
+                        If the ``primary`` key is defined, but not the
+                         ``security`` key, then then configuration for
+                         ``primary`` is also used for ``security``.
+                        If ``search_dns`` is used for the ``security``
+                         key, the search pattern will be
+                         ``<distro>-security-mirror``.
+
+                        If no mirrors are specified, or all lookups fail,
+                        then default mirrors defined in the datasource
+                        are used. If none are present in the datasource
+                        either the following defaults are used:
+
+                        ``primary`` => ``http://archive.ubuntu.com/ubuntu``,
+                         ``security`` =>
+                         ``http://security.ubuntu.com/ubuntu``
+                        """)},
+                'security': {
+                    **mirror_property,
+                    'description': dedent("""\
+                        Please refer to the primary config documentation""")
+                },
+                'add_apt_repo_match': {
+                    'type': 'string',
+                    'description': dedent("""\
+                        All source entries in ``apt-sources`` that match
+                         regex in ``add_apt_repo_match`` will be added to
+                         the system using ``add-apt-repository``. If
+                          ``add_apt_repo_match`` is not specified, it
+                         defaults to ``^[\\w-]+:\\w``""")
```

Comment:
Please use the global ADD_APT_REPO_MATCH from the file instead of repeating in docs (in case it changes)

It's probably worth adding a 'default': ADD_APT_REPO_MATCH, here too to the schema

Reply:
Done